### PR TITLE
More readable search errors

### PR DESCRIPTION
**Pull request by zlayne**
Signed-off-by: Zack Layne <zlayne@redhat.com>

Related issues: 
- https://github.com/stolostron/backlog/issues/24788
- https://github.com/stolostron/backlog/issues/22391

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -89,6 +89,7 @@
     "Already enforcing": "Already enforcing",
     "Already informing": "Already informing",
     "Amazon Web Services": "Amazon Web Services",
+    "An error occurred while contacting the search service.": "An error occurred while contacting the search service.",
```
Comment (jlpadilla) [inline frontend/public/locales/en/translation.json:92]: For consistency, I see that most other errors don't start with "An ..."

File: frontend/src/routes/Home/Search/SearchPage.tsx [hunk 2]
```diff
@@ -48,29 +49,39 @@ const useStyles = makeStyles({
...
-                    }
-                    subtitle={schemaError?.message || completeError?.message}
-                />
-            </div>
+            <EmptyState>
+                <EmptyStateIcon icon={InfoCircleIcon} color={'var(--pf-global--info-color--100)'} />
+                <Title size="lg" headingLevel="h4">
+                    {t('search.filter.info.title')}
+                </Title>
+                <EmptyStateBody>{schemaError?.message || completeError?.message}</EmptyStateBody>
```
Comment (jlpadilla) [inline frontend/src/routes/Home/Search/SearchPage.tsx:62]: Do we always have one of these 2 error messages? I wonder if we should add `... || 'Default empty state message'`

Comment (zlayne) [reply frontend/src/routes/Home/Search/SearchPage.tsx:62]: We will always have one or the other here - the conditional above checks that at least one is defined.

**Pull request merged**